base/exports.jl

```diff
@@ -950,6 +950,7 @@ export
     sizeof,
 
 # tasks and conditions
+    channeled_tasks,
```

Comment:
This was inspired by the usage in `test/worlds.jl` which used produce/consume in a bi-directional communication way (which would need to be simulated by 2 channels). I think a common case would actually be starting a bunch of worker tasks reading of an input channel and writing to an output channel driven.